### PR TITLE
chore(storybook): Upgrade to storybook 6 and deps update - front 1482

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: php/ec/.storybook/main.js [hunk 1]
```diff
@@ -12,11 +14,13 @@ const addons = [
...
 
-const managerWebpack = async baseConfig => {
-  // Exclude node_modules
-  baseConfig.module.rules[0].exclude = /node_modules\/(?!@ecl-twig\/).*/;
-
-  return baseConfig;
+module.exports = {
+  stories,
+  addons,
+  webPackFinal: (config) => {
```
Comment (kalinchernev) [inline php/ec/.storybook/main.js:20]: I haven't had the opportunity to debug this particular part of the config, but seems to me the exclude rule is getting lost with the updates.

Are you sure webpack.config.js is meant to replace this block?

Comment (planctus) [reply php/ec/.storybook/main.js:20]: True, good that you noticed this, basically the webpack.config.js was still acting, but the final "merge" in the main.js export was not good, i copied that from here:  https://storybook.js.org/docs/react/configure/webpack#using-your-existing-config
It seems enough to just move the customisation we make directly in the main.js in this webpackFinal key, instead, so i went for that and removed the webpack.config.js

**Comment by planctus** [comment]
@kalinchernev, conflicts fixed ;)

**Review by kalinchernev — APPROVED** [review]
Changes look ok and testing the preview version of storybook items work as expected.
CI should be ok and I'll merge it

**Pull request merged**